core/src/main/scripts/importer/validateData.py

```diff
@@ -1248,8 +1248,19 @@ def checkSwissProt(self, value):
         else:
             # format described on http://www.uniprot.org/help/entry_name
             if not re.match(
-                        r'^[A-Z0-9]{2,5}_[A-Z0-9]{2,5}$',
+                        r'^[A-Z0-9]{1,5}_[A-Z0-9]{1,5}$',
                         value):
+                value_list = value.strip('"').split(',')
```

Comment:
we can simplify this whole part by just letting the whole scenario return a warning. So if the regular expression does not match, then there should be a warning about invalid name.